### PR TITLE
Pipeline Runs Event Descriptions

### PR DESCRIPTION
**Pull request by aubuchcl**
Two files change to make run events correct about timestamps

### CODE REVIEW

File: components/schemas/pipelines/runs/PipelineRun.yml [hunk 1]
```diff
@@ -45,7 +45,22 @@ properties:
...
+          description: A collection of timestamps for each event in the pipeline run's lifetime.
+          required:
+            - started
+            - queued
+            - finished
+          properties:
+            started:
+              description: The timestamp of when the pipeline stage was started.
+              "$ref": "../../DateTime.yml"
+            queued:
```
[the remark this answers is not in the document]
Comment (aubuchcl) [reply components/schemas/pipelines/runs/PipelineRun.yml]: should be resolved

File: components/schemas/pipelines/runs/PipelineRunStep.yml [hunk 2]
```diff
@@ -39,7 +39,22 @@ properties:
...
+    description: A collection of timestamps for each event in the pipeline run's lifetime.
+    required:
+      - started
+      - queued
+      - finished
+    properties:
+      started:
+        description: The timestamp of when the pipeline step was started.
+        "$ref": "../../DateTime.yml"
+      queued:
```
[the remark this answers is not in the document]
Comment (aubuchcl) [reply components/schemas/pipelines/runs/PipelineRunStep.yml]: resolved

File: components/schemas/pipelines/runs/PipelineRun.yml [hunk 3]
```diff
@@ -45,7 +45,19 @@ properties:
           items:
             $ref: ./PipelineRunStep.yml
         events:
-          $ref: ./PipelineRunEvents.yml
+          type: object
+          description: A collection of timestamps for each event in the pipeline run's lifetime.
+          required:
+            - started
+            - queued
```
Comment (blewisCycle) [inline components/schemas/pipelines/runs/PipelineRun.yml:52]: Can't be required if it doesn't exist in the properties

File: components/schemas/pipelines/runs/PipelineRunStep.yml [hunk 4]
```diff
@@ -39,7 +39,19 @@ properties:
       - webhook.post
       - webhook.get
   events:
-    $ref: ./PipelineRunEvents.yml
+    type: object
+    description: A collection of timestamps for each event in the pipeline run's lifetime.
+    required:
+      - started
+      - queued
```
Comment (blewisCycle) [inline components/schemas/pipelines/runs/PipelineRunStep.yml:46]: Same

**Review by blewisCycle — APPROVED** [review]

**Pull request merged**